sympy/functions/elementary/exponential.py

```diff
@@ -788,7 +789,7 @@ def _eval_expand_log(self, deep=True, **hints):
             expr = []
             nonpos = []
             for x in arg.args:
-                if force or x.is_positive or x.is_polar:
+                if force or not x.is_negative or x.is_polar:
```

Comment:
Why do we need to change this?

Reply:
Basically this allows for x to be 0 when deciding if it can expand log using the product rule.

log(x*y) = log(x) + log(y)

This came up in my tests because I want

log(x**2 * y**4) = 2log(x) + 4log(y)

when x and y are real. But x and y being real only guarantees that x**2 and y**4 are non-negative (could be 0), whereas the condition was checking for is_positive. So instead I check for not is_negative.